### PR TITLE
chore: code cleanup services/web

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
:broom: 

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by micbar** [comment]
@fschade please check that out. You are refactoring a lot in the web service.

Please make sure that you do not overlap 😬 

just saying ...

File: services/web/pkg/apps/apps.go [hunk 1]
```diff
@@ -3,11 +3,11 @@ package apps
 import (
 	"encoding/json"
 	"errors"
+	"github.com/go-playground/validator/v10"
```
Comment (micbar) [inline services/web/pkg/apps/apps.go:6]: that is not correct. Top is only for the std lib.

File: services/web/pkg/apps/apps.go [hunk 2]
```diff
@@ -129,6 +123,7 @@ func Build(fSystem fs.FS, id string, conf map[string]any) (Application, error) {
 		return Application{}, errors.Join(err, ErrInvalidManifest)
 	}
 
+	validate := validator.New(validator.WithRequiredStructEnabled())
```
Comment (fschade) [inline services/web/pkg/apps/apps.go:126]: can you explain why we do not have it global anymore? not a fan to re-initialize it on every build call

File: services/web/pkg/service/v0/service.go [hunk 3]
```diff
@@ -165,7 +164,7 @@ func (p Web) Static(f fs.FS, root string, ttl int) http.HandlerFunc {
 			return
 		}
 
-		w.Header().Set("Cache-Control", fmt.Sprintf("max-age=%s", strconv.Itoa(ttl)))
+		w.Header().Set("Cache-Control", "max-age="+strconv.Itoa(ttl))
```
Comment (fschade) [inline services/web/pkg/service/v0/service.go:167]: fmt.Sprintf is slower compared to string concatenation

File: services/web/pkg/apps/apps.go [hunk 4]
```diff
@@ -107,7 +101,7 @@ func List(logger log.Logger, data map[string]config.App, fSystems ...fs.FS) []Ap
 	return maps.Values(registry)
 }
 
-func Build(fSystem fs.FS, id string, conf map[string]any) (Application, error) {
+func build(fSystem fs.FS, id string, conf map[string]any) (Application, error) {
```
Comment (fschade) [inline services/web/pkg/apps/apps.go:104]: not against it to make it private, but may i ask why?

**Review by fschade — CHANGES_REQUESTED** [review]

File: services/web/pkg/apps/apps.go (hunk 4 above)
Comment (DeepDiver1975) [reply services/web/pkg/apps/apps.go:104]: It is not used outside of the package.

File: services/web/pkg/service/v0/service.go (hunk 3 above)
Comment (DeepDiver1975) [reply services/web/pkg/service/v0/service.go:167]: Yes.

File: services/web/pkg/apps/apps.go (hunk 2 above)
Comment (DeepDiver1975) [reply services/web/pkg/apps/apps.go:126]: Init function is discouraged.
Is the validator.new expensive?

**Comment by micbar** [comment]
@DeepDiver1975 @fschade

Please get it merged or close.

**Comment by fschade** [comment]
> @DeepDiver1975 @fschade
> 
> Please get it merged or close.

i take care, thx

**Comment by DeepDiver1975** [comment]
@fschade I can do the rebase ....

**Comment by fschade** [comment]
> @fschade I can do the rebase ....

some things changed in the meantime, the branding is now its own package, backported. Waiting for the ci now.

**Review by fschade — APPROVED** [review]

**Comment by DeepDiver1975** [comment]
Nice! THX

**Pull request merged**